src/components/command-bar/commands/search/unified-search/components/link-region/types.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import { LinkProps } from 'components/link'
+import { ReactNode } from 'react'
+
+type InheritedLinkProps = Pick<LinkProps, 'href' | 'onClick' | 'opensInNewTab'>
+
+export interface LinkRegionProps extends InheritedLinkProps {
+	/**
+	 * Children to render in the linked region.
+	 */
+	children: ReactNode
```

Comment:
💭 you can get out-of-the-box `children` typing with `PropsWithChildren` from react!

2 usages:
- either `MyInterface extends PropsWithChildren`
- or `PropsWithChildren<MyInterface>`

Reply:
✅ great tip, TIL! 🙌 updated to do the `PropsWithChildren<MyInterface>` - type approach!